andela/ah-backend-dojo
# #165273593 User sees read statistics

[Finishes #165273593]
#### What does this PR do?
Adds reading stats to an article

#### Description of the task to be completed?
Add unit test for reading stats.
Add functionality for reading stats

#### Any background context you want to provide?
A reader can only add to the read stats on an article once hence viewing an article twice does not increase its view by two.
A reader can not add to the read stats of their own article.

#### How should this be manually tested?
View an article you have created and check on the statistics. They should not change
View an article created by another user and check on its statistics again.

#### What are the relevant pivotal tracker stories
[#165273593](https://www.pivotaltracker.com/story/show/165273593)

#### Postman screenshots
Get a specific article.
<img width="1440" alt="Screenshot 2019-05-06 at 22 02 34" src="https://user-images.githubusercontent.com/24764476/57248327-c49c9e00-704a-11e9-8087-18eec5ca6c03.png">
